website/docs/getting-started/quick-start.mdx

```diff
@@ -299,83 +315,144 @@ lockScript:
 </Tabs>
 ```
 
-Check out specific account's balance:
+## Deposits and Balance Checks
+
+:::note
+Make sure you have run `offckb node` before doing the following.
```

Comment:
```suggestion
Make sure you have run `offckb node` before doing the following targeting devnet.
```

something like this? targeting testnet doesn't required local node running.

Reply:
updated in [commit](https://github.com/nervosnetwork/docs.nervos.org/pull/331/commits/3e194fd9d37726a73d04334b1aa8fcadb66cf6f6)